File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1013.cpp
#include <stdio.h>
#include <stdlib.h>

int main(){
    int a = 0;
    int b = 0;
    int c = 0;
    int maiorAB = 0;
    int maior = 0;
    
    scanf("%i", &a);
    scanf("%i", &b);
    scanf("%i", &c);
    
    maiorAB = (a + b + abs(a-b))/2;
    maior = (maiorAB + c +abs(maiorAB - c))/2;
    
    printf("%i eh o maior\n", maior);
    
    return 0;
}

File: Uri-Online-Judge/Wellington/Python 3/Iniciante/1013.py
# -*- coding: utf-8 -*-

linha = list(map(int, input().split()))

a, b, c = linha

maiorAB = int((a + b + abs(a-b))/2)
maior = int((maiorAB + c + abs(maiorAB - c))/2)

print('{} eh o maior'.format(maior))